includes/admin/class-wp-job-manager-settings.php

```diff
@@ -635,7 +691,19 @@ protected function input_hidden( $option, $attributes, $value, $placeholder ) {
 	 * @param string $placeholder
 	 */
 	protected function input_password( $option, $attributes, $value, $placeholder ) {
-		?><input id="setting-<?php echo esc_attr( $option['name'] ); ?>" class="regular-text" type="password" name="<?php echo esc_attr( $option['name'] ); ?>" value="<?php echo esc_attr( $value ); ?>" <?php echo implode( ' ', $attributes ); ?> <?php echo $placeholder; ?> /><?php
+		?>
+		<input
+			id="setting-<?php echo esc_attr( $option['name'] ); ?>"
+			class="regular-text"
+			type="password"
+			name="<?php echo esc_attr( $option['name'] ); ?>"
+			value="<?php echo esc_attr( $value ); ?>"
+			<?php
+			echo implode( ' ', $attributes ) . ' '; // WPCS: XSS ok.
+			echo $placeholder; // WPCS: XSS ok.
```

Comment:
As discussed, these are escaped in the method that calls these methods. I considered a refactor, but might save it for a different PR as it might break functionality.